torch/_decomp/decompositions.py

```diff
@@ -4440,7 +4440,7 @@ def squeeze_default(self: Tensor, dim: Optional[int] = None):
 
 
 @register_decomposition(torch.ops.aten._weight_norm_interface)
-def _weight_norm_interface(x, y, dim):
+def _weight_norm_interface(x, y, dim=0):
```

Comment:
Looks fine, but can you add a test for it?

Reply:
will do! thanks for the review